src/assets/scripts/client/aircraft/FlightManagementSystem/WaypointModel.js

```diff
@@ -188,6 +188,33 @@ export default class WaypointModel {
     }
 
     /**
+     * @for WaypointModel
+     * @property hasAltitudeRestriction
+     * @type {boolean}
+     */
+    get hasAltitudeRestriction() {
+        return this.altitudeMaximum !== -1 || this.altitudeMinimum !== -1;
+    }
+
+    /**
+     * @for WaypointModel
+     * @property hasRestriction
+     * @type {boolean}
+     */
+    get hasRestriction() {
```

Comment:
This getter should be below `hasSpeedRestriction`.  I know you want alphabetical but it's best practice to [usually] define a method before it's used.  Additionally, I'm not so sure these are getters. why did you decide to use getters here instead of methods?